dapp/src/components/dashboard/containers/MainDashboard/index.jsx

```diff
@@ -57,11 +56,17 @@ class DashboardLayout extends Component {
       if (window && window.analytics && isAdmin) {
         const { analytics } = window
 
-        analytics.identify(`${accountAddress}`, {
-          role: 'admin'
-        })
-
         if (location.pathname.includes('/justCreated')) {
+          analytics.ready(() => {
```

Comment:
Two questions:
- Maybe we can move this logic out of component? to redux-saga by example. So we don't need the /justCreated parameter.
- If I issued one community, used the bridg (bridgeWasUsed is true). Then created a new one, `bridgeWasUsed` would be false then?

Reply:
1. It's good advice. I'll change it.
2. I thought so but it turns out it doesn't work that way for some reason. I am still investigating the issue